plugins/woocommerce-blocks/assets/js/blocks/product-collection/constants.ts

```diff
@@ -1,10 +1,7 @@
 /**
```

Comment:
As suggested above, if you rename `constants-register-product-collection.ts` -> `constants.ts`, this one should become `utils.ts` or something similar? WDYT?